src/acrcssc/azext_acrcssc/helper/_taskoperations.py

```diff
@@ -43,41 +43,45 @@
 logger = get_logger(__name__)
 
 
-def create_update_continuous_patch_v1(cmd, registry, cssc_config_file, schedule, dryrun, run_immediately, is_create_workflow=True):
+def create_update_continuous_patch_v1(cmd,
+                                      registry,
+                                      cssc_config_file,
+                                      schedule,
+                                      dryrun,
+                                      run_immediately,
+                                      is_create_workflow=True):
+
     logger.debug(f"Entering continuousPatchV1_creation {cssc_config_file} {dryrun} {run_immediately}")
+
     resource_group = parse_resource_id(registry.id)[RESOURCE_GROUP]
     schedule_cron_expression = None
     if schedule is not None:
         schedule_cron_expression = convert_timespan_to_cron(schedule)
+
     logger.debug(f"converted schedule to cron expression: {schedule_cron_expression}")
-    cssc_tasks_exists = check_continuous_task_exists(cmd, registry)
+
+    task_list = []
+    cssc_tasks_exists = check_continuous_task_exists(cmd, registry, task_list=task_list)
```

Comment:
Suggest, instead of passing task_list as a function argument, return it as part of the result, as modifying a mutable argument inside the function can sometimes be less readable.
E.g.
cssc_tasks_exists, task_list = check_continuous_task_exists(cmd, registry)